### PR TITLE
Add MM build target to Jenkinsfile

### PR DESCRIPTION
**Pull request by AngheloAlf**
Jenkins now should checkout the latest MM and test it, so we don't broke compatibility with them.

Issue: #135

### CODE REVIEW

**Comment by AngheloAlf** [comment]
Looks like MM failed to build because of some issues with the `OutputFormatter`

There is output like this in MM scenes with current ZAPD's master:
```c
static SCmdCsCameraList Z2_SYATEKI_MORISet0000Cmd0D = { 0x02, 1, (u32)&Z2_SYATEKI_MORICsCameraList0x000574 }; // 0x0068
    }; 

static SCmdMinimapSettings Z2_SYATEKI_MORISet0000Cmd0E = { 0x1C, 0, (u32)&Z2_SYATEKI_MORIMinimapList0x000558 }; //
    0x0070 }; 
```

Problems like this are fixed in #124. but that branch wont build properly either because of the lack of the new macros for scenes commands (https://github.com/zeldaret/mm/pull/114).

So I'll convert this PR into a draft until those two PRs get merged in their respective repos.

File: Jenkinsfile [hunk 1]
```diff
@@ -38,6 +38,35 @@ pipeline {
...
+            }
+        }
+        stage('Set up mm') {
+            steps {
+                dir('mm') {
+                    sh 'cp /usr/local/etc/roms/mm.us.rev1.z64 baserom.z64'
+
+                    // Identical to `make setup` except for copying our newer ZAPD.out into oot
+                    sh 'git submodule update --init --recursive'
+                    sh 'python3 -m pip install -r requirements.txt'
```
Comment (ethteck) [inline Jenkinsfile]: We shouldn't be installing any dependencies at build time. What's missing that requires this?

Comment (AngheloAlf) [reply Jenkinsfile]: I copy-pasted that from MM's makefile. From the setup step to be precise.
https://github.com/zeldaret/mm/blob/aae6d9d3644f0792a27b3c67758345c6d13e909b/Makefile#L196

I guess I should remove this, right?

Comment (ethteck) [reply Jenkinsfile]: Ah okay. My bad for missing it there then. Yeah I think it can be removed

**Comment by AngheloAlf** [comment]
MM updated their ZAPD version, so now this properly works.
Would be nice to prioritize this PR to prevent breaking MM compatibility in the future.

**Review by fig02 — APPROVED** [review]

**Review by NEstelami — APPROVED** [review]
LGTM

**Pull request merged**